pkg/aks/create.go

```diff
@@ -236,6 +236,13 @@ func createManagedCluster(ctx context.Context, cred *Credentials, workplacesClie
 		managedCluster.APIServerAccessProfile.EnablePrivateCluster = spec.PrivateCluster
 	}
 
+	if &cred.TenantID != "" {
```

Comment:
@MAXxATTAXx you meant:

```suggestion
	if &cred.TenantID != nil {
```

perhaps, based on the CI [failure](https://drone-pr.rancher.io/rancher/aks-operator/461/1/2) [`pkg/aks/create.go:250:23: invalid operation: &cred.TenantID != "" (mismatched types *string and untyped string)`
]

Reply:
THanks for pointing that out. Submitted changes to address this.